test/Microsoft.ML.Benchmarks.Tests/BenchmarksTest.cs

```diff
@@ -39,7 +39,11 @@ public class BenchmarksTest
 
         private ITestOutputHelper Output { get; }
 
+#if DEBUG
         [Fact(Skip = SkipTheDebug)]
+#else
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]
```

Comment:
Do we have a bug logged for this?

IMO - it is good practice to link the bug in the code for cases like this. So someone who is reading the code can easily find the bug tracking this, and see what the current status is.

Reply:
We have a bug filed for this #1391, I will add a reference to it in my next PR. Thanks for the comment!